<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remark_dishes</class>
 <widget class="QDialog" name="Remark_dishes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>721</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>721</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Remark_dishes{
border-image:url(:/Image/登录界面.png);

}



</string>
  </property>
  <widget class="QPushButton" name="star5_pbt_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="waiter_textEdit_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>270</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(244,244,244,100);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="listView_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>135</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 100);
border-radius:0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star1_pbt_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star4_pbt_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star3_pbt_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>菜品名字：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="star2_pbt_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="listView_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>285</y>
     <width>295</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 100);
border-radius:0;</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>215</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 100);
border-radius:0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>顾客ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
